testsuite/src/commonMain/kotlin/org/jetbrains/litmuskt/tests/UnsafePublication.kt

```diff
@@ -66,19 +66,42 @@ object UnsafePublication {
         }
     }
 
-    val PlainArray = litmusTest({
+    val PlainArrayBoxed = litmusTest({
         object : LitmusIOutcome() {
             var arr: Array<Int>? = null
         }
     }) {
         thread {
-            arr = Array(10) { 0 }
+            arr = Array(10) { 1 }
```

Comment:
Let's use single element array in all tests for consistency:
```suggestion
            arr = Array(1) { 1 }
```